radanalyticsio/spark/cluster/base/role.yaml

```diff
@@ -0,0 +1,18 @@
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: edit-resources
+  labels:
+    app: jupyterhub
+    component: spark-operator
+  namespace: "{{ meta.namespace }}"
```

Comment:
iirc, you can leave the namespace out and kfctl will automatically apply the correct namespace based on where the kfdef exists